src/compiler/GF/Server.hs

```diff
@@ -6,11 +6,12 @@ import qualified Data.Map as M
 import Control.Applicative -- for GHC<7.10
 import Control.Monad(when)
 import Control.Monad.State(StateT(..),get,gets,put)
-import Control.Monad.Error(ErrorT(..),Error(..))
+--import Control.Monad.Error(ErrorT(..),Error(..))
+import Control.Monad.Except
```

Comment:
Not sure if it's significant, but you could follow the pattern and do explicit imports:

```haskell
import Control.Monad.Except(ExceptT,runExceptT)
```
(if it doesn't work, try importing `ExceptT(..)`)